index.ch.md

```diff
@@ -0,0 +1,21 @@
+_2021-03-23_
+
+Richard M. Stallman(常被称为RMS)，很长时间以来在自由软件社区内都是一股危险的力量。他过去表现出了很多不当行为，比如歧视女性、歧视残障人士、歧视跨性别人群。这些行为不应该在自由软件、数字权利和技术社区中立足。随着他最近重新回到自由软件基金董事会，我们呼吁FSF董事会下台，并要求撤销RMS的所有领导职位。
```

Comment:
```suggestion
Richard M. Stallman(常被称为RMS)，很长时间以来在自由软件社区内都是一股危险的力量。他过去表现出了很多不当行为，比如歧视女性、残障人士与跨性别人群。这些行为不应该在自由软件、数字权利和技术社区中存在。他最近重新回到了自由软件基金董事会，我们呼吁FSF董事会下台，并要求撤销RMS的所有领导职位。
```

Reply:
Try to read your words aloud, if you write awkward sentences, you'll feel twice as awkward reading it.